spacy/tokens/doc.pyx

```diff
@@ -1461,16 +1526,14 @@ cdef class Doc:
         """
         data = {"text": self.text}
         if self.has_annotation("ENT_IOB"):
-            data["ents"] = [{"start": ent.start_char, "end": ent.end_char,
-                            "label": ent.label_} for ent in self.ents]
+            data["ents"] = [{"start": ent.start_char, "end": ent.end_char, "label": ent.label_} for ent in self.ents]
         if self.has_annotation("SENT_START"):
             sents = list(self.sents)
-            data["sents"] = [{"start": sent.start_char, "end": sent.end_char}
-                             for sent in sents]
+            data["sents"] = [{"start": sent.start_char, "end": sent.end_char} for sent in sents]
         if self.cats:
             data["cats"] = self.cats
         data["tokens"] = []
-        attrs = ["TAG", "MORPH", "POS", "LEMMA", "DEP"]
+        attrs = ["TAG", "MORPH", "POS", "LEMMA", "DEP", "SENT_START"]
```

Comment:
Let's not duplicate the sentence annotation, since there's some chance it will get out-of-sync. Let's keep `data["sents"]` above with character offsets and then convert them as necessary.

Reply:
Removed in next commit.